keyboards/converter/ps2_usb/matrix.c

```diff
@@ -0,0 +1,446 @@
+/*
+Copyright 2011 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "action.h"
+#include "print.h"
+#include "util.h"
+#include "debug.h"
+#include "ps2.h"
+#include "host.h"
+#include "led.h"
+#include "matrix.h"
+
+static void matrix_make(uint8_t code);
+static void matrix_break(uint8_t code);
+
```

Comment:
To fix at least one of the compilation errors, move `matrix_clear` higher up in the file, or add this here: 
```suggestion
void matrix_clear(void);

```